.github/workflows/release.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
chanesets/action uses existing `.npmrc` if it does. [(doc)](https://github.com/changesets/action#:~:text=However%2C%20if%20a%20.npmrc%20file%20is%20found%2C%20the%20GitHub%20Action%20does%20not%20recreate%20the%20file)

So using `NODE_AUTH_TOKEN` to follow setup-node here.